privaterelay/locales/en-US/app.ftl

```diff
@@ -94,20 +90,20 @@ hero-image-copy-control-html = Now <em>you’re in control</em> of what’s arri
 
 faq-headline = Frequently Asked Questions
 faq-question-1-question = What about spam?
-faq-question-1-answer-a-html = While { -brand-name-relay } does not filter for spam, our email partner Amazon SES does block spam and malware. If { -brand-name-relay } forwards messages you don’t want, you can update your { -brand-name-relay } settings to block messages from the alias forwarding them.  
+faq-question-1-answer-a = While { -brand-name-relay } does not filter for spam, our email partner Amazon SES does block spam and malware. If { -brand-name-relay } forwards messages you don’t want, you can update your { -brand-name-relay } settings to block messages from the alias forwarding them.  
 
 # Variables:
 #   $url (url) - https://addons.mozilla.org/firefox/addon/private-relay/
 #   $attrs (string) - specific attributes added to external links
-faq-question-1-answer-b-html = If you see a broader problem of unwanted email from all of your aliases, please <a href="{ $url }" { $attrs }>report this to us</a> so we can consider adjusting the SES spam thresholds for this service. If you report these as spam, your email provider will see { -brand-name-relay } as the source of spam, not the original sender.
+faq-question-1-answer-b-html = If you see a broader problem of unwanted email from all of your aliases, please <a href={ $url } { $attrs }>report this to us</a> so we can consider adjusting the SES spam thresholds for this service. If you report these as spam, your email provider will see { -brand-name-relay } as the source of spam, not the original sender.
```

Comment:
Why is this removing the `"` chars?

Reply:
The original intent was to make it render correctly (as it was adding quotes already), however – that is not the case. I'll add them back.